wiki/Beatmapping/Beats_per_minute/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - BPM
+  - Tempo
+---
+
+# Beats per minute
+
+**Beats Per Minute**, abbreviated as *BPM*, refers to the number of whole beats in a minute. BPM is used to measure the [tempo](wikipedia.org/wiki/Tempo) of a song. This is needed to line up the beats of a song with the [hit objects](/wiki/Hit_Objects) of a [beatmap] successfully. It can be changed in the [beatmap editor](/wiki/Beatmap_Editor) under the `Timing` tab, or by using the shortcut (`Ctrl` + `P`).
```

Comment:
```suggestion
**Beats per minute**, abbreviated as *BPM*, refers to the number of whole beats in a minute. This is used to measure the [tempo](https://en.wikipedia.org/wiki/Tempo) of a song, which is then used to place [hit objects](/wiki/Hit_Objects) in a [beatmap](/wiki/Beatmap). Setting the BPM is the second thing that needs to be done when creating a new beatmap—the first being [song setup](/wiki/Beatmap_Editor/Song_Setup). It can be changed later in the [beatmap editor](/wiki/Beatmap_Editor) under the `Timing` tab, or by pressing `Ctrl` + `P`.
```